homeassistant/components/synology_dsm/coordinator.py

```diff
@@ -131,8 +124,7 @@ async def _async_update_data(self) -> dict[str, dict[str, SynoCamera]]:
         }
 
         try:
-            async with async_timeout.timeout(30):
-                await self.hass.async_add_executor_job(surveillance_station.update)
+            await surveillance_station.update()
```

Comment:
Is the timeout built-in now?

Reply:
there is one timeout in general for every api-call used:
https://github.com/mib1185/py-synologydsm-api/blob/18869e1aa940b1f267701fcd0dc4fbc2f382c51a/src/synology_dsm/synology_dsm.py#L285-L288

https://github.com/mib1185/py-synologydsm-api/blob/18869e1aa940b1f267701fcd0dc4fbc2f382c51a/src/synology_dsm/synology_dsm.py#L297-L298